frontend/src/modules/app/ToolBar.tsx

```diff
@@ -248,6 +299,9 @@ export const ToolBar = ({
     initParameterUpdate();
   }, [ventilating, initParameterUpdate]);
 
+  /**
+   * Update Label on Button based on ventilation status
+   */
   useEffect(() => {
     if (ventilating) {
       history.push(DASHBOARD_ROUTE.path);
```

Comment:
(line 323) What is `handleOnClick` supposed to do when it's called? (@rohanpurohit was this something you added, or was it there before?)

(line 353, 357) Same questions for `handleDiscardClose` and `handleDiscardConfirm` - @rohanpurohit was this code you introduced? Why does this code need to be in `ToolBar.tsx` / what is the purpose of the alarm limits discard modal in the toolbar?